static/scss/partials/main.scss

```diff
@@ -1115,6 +1121,116 @@ firefox-private-relay-addon[data-addon-installed="true"] + firefox-private-relay
     background-image: url("/static/images/logos/mpp-temp-icon.png");
 }
 
+.c-data-notification-banner {
+    display: flex;
+    flex-direction: column;
+    background-color: $color-white;
+    box-shadow: $box-shadow-sm;
+    border-radius: $border-radius-md;
+
+    .data-notification-banner-bg {
+        padding: $spacing-xs;
+        width: 100%;
+        position: relative;
+        min-height: 0;
+    }
+
+    .data-notification-banner-content {
+        padding-left: $spacing-sm;
+        display: flex;
+        flex-direction: row;
+    }
+
+    .data-notification-banner-copy {
+        display: flex;
+        flex-direction: column;
+        padding: $spacing-lg $spacing-lg $spacing-md $spacing-lg;
+    }
+
+    .data-notification-banner-header {
+        font-family: $font-metropolis;
+        font-size: 1.125em;
+        font-weight: 700;
+        text-align: center;
+        width: 90%;
+        padding-bottom: $spacing-sm;
+        margin: 0;
+    }
+
+    .data-notification-banner-header:before {
+        content: "";
+        display: inline-block;
+        height: 16px;
+        width: 16px;
+        margin-right: 12px;
+        background-image: url("/static/images/icon-orange-info.svg");
+        background-size: 16px;
+        background-repeat: no-repeat;
+        background-position: center center;
+    }
+
+    .data-notification-banner-sub {
+        margin: 0;
+        line-height: 150%;
+    }
+
+    .data-notification-banner-link {
+        text-align: center;
+        color: $color-blue-50;
+        margin: 0;
+        font-size: 1rem;
+        font-weight: 600;
+        padding: $spacing-lg 0;
+    }
+
+    .dismiss-wrapper {
+        width: 100%;
+        height: 0px;
+    }
+
+    .data-notification-dismiss {
+        background: none;
+        cursor: pointer;
+        display: block;
+        border: none;
+        float: right;
+        position: relative;
+        z-index: 1;
+        padding: $spacing-md $spacing-md 0 0;
+
+        &:hover {
+            transform: scale(1.05);
+        }
+        
+        img {
+            width: 1.5em;
+        }
+    }
+
+    @media #{$mq-md} {
+
+        .data-notification-banner-copy:before {
+            content: '';
+            position: absolute;
+            z-index: 1;
+            height: calc(100% - #{$spacing-lg});
+            margin-top: -$spacing-lg;
+            width: 5px;
+            margin-left: -$spacing-2xl;
+            background: $color-yellow-50;
+            border-radius: 5px;
+        }
+
+        .data-notification-banner-copy {
+            padding: $spacing-xl $spacing-xl $spacing-md $spacing-2xl;
+        }
+
+        .data-notification-banner-header, .data-notification-banner-link {
+            text-align: left;
+        }
```

Comment:
Will this work for RTL languages? Or is the banner only for English/LTR languages?

Reply:
@pdehaan We don't have any considerations for RTL languages right now. This may warrant a follow-up issue altogether. I'm not even sure we're localizing in a RTL language currently. 

@peiying2 Can you confirm if we're translating any RTL languages? 

@codemist Does this work without the `text-align: left` designation?

Reply:
It's defaulted to `text-align: center;` for mobile use case, and changes to left on larger devices, since Protocol goes by `min-width` for media queries (correct me if I'm wrong).

Reply:
@pdehaan Does that work for you? Is there something else we should do?